### ISSUE TITLE
Using scss additionalData to import results in an error

### ISSUE THREAD

**Issue by akhlivnoy** [mrmckeb/typescript-plugin-css-modules#199]
**Description**
---
Adding @import .scss common file using Vite defineConfig leads to error "Property 'prop_name' does not exist on type '{}'."

**To Reproduce**
---
1. Add to **`defineConfig`** next options (no matter alias, absolute or relative path):
```
defineConfig({
    ...,
    css: {
      modules: {
        localsConvention: 'camelCaseOnly',
      },
      preprocessorOptions: {
        scss: {
          additionalData: "@import 'src/app/styles/variables';",
        },
      },
    },
  });
```
2. Use any variable from imported file in any another scss file.

**Expected behavior**
---
![image](https://user-images.githubusercontent.com/42786504/217334811-11a2acbf-1cab-4dad-ba09-29af26785fdb.png)

**But got**
---
![image](https://user-images.githubusercontent.com/42786504/217335305-c54c5016-9691-41bd-b115-d5b918077ad7.png)

**Note that before the variable is used, everything is fine**

**Also**
---
Using `@import 'styles/variables';` manually in file solves the problem, but only using alias path
**`vite.config.ts`**
```
defineConfig({
    ...,
    resolve: {
      alias: {
        styles: path.resolve(__dirname, './src/app/styles'),
      },
    },
  });
```
**`tsconfig.json`**
```
{
  "compilerOptions": {
    ...,
    "paths": {
      "styles": ["app/styles"],
      "styles/*": ["app/styles/*"],
    },
    "plugins": [
      {
        "name": "typescript-plugin-css-modules",
        "options": {
          "classnameTransform": "camelCaseOnly",
        }
      }
    ]
  }
}
```

**Comment by mrmckeb** [comment]
Hi @akhlivnoy,

We do have the ability to pass options through to processors, but it looks like Vite isn't doing that - this is actually a custom Vite feature (also available via some Webpack loaders).

I'll mark this as a feature request, and I think I should be able to deliver it shortly.

**Comment by mrmckeb** [comment]
This is now available in `v4.2.1`:
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.1
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.0

**Comment by chenbingchang** [comment]
> 现在可用于`v4.2.1`：
> 
> * https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.1
> * https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.0

Hello, the problem does not seem to be resolved
![image](https://github.com/mrmckeb/typescript-plugin-css-modules/assets/43459077/71f84352-d3c3-4165-96eb-077c394c75ff)
